mnist/tensorflow/mnist_with_summaries/README.md

```diff
@@ -139,3 +139,35 @@ mnist-cos-train-tf   2022-01-04T08:27:29Z
 **NOTICE** If github repo can't be used, you can change codeset git url to `https://gitee.com/xieyuandong/tf-mnist.git` and change train job command to `python /root/tf-mnist/mnist_with_summaries_savedmodel.py --log_dir=/tmp/xieyuandong/mnist-cos-train-tf/logs --learning_rate=0.01 --batch_size=150 --data_dir=/cos/MNIST/raw/ --saved_model_dir=/cos/mnist_seldon`.
 
 ## Inference
+
+### Install Seldon with Istio
+
+```shell
+$ kubectl create namespace istio-system
+```
+
+Using TKE Mesh create service mesh. notice set name `seldon-gateway` in `istio-system`, create ingress gateway watch default namespace application. Then create Gateway `seldon-gateway` in default namespace and set port is 8080
```

Comment:
Can you please explain more about it? I do not know how to do it according to the doc.